src/modules/PowerShell/FinOpsHub/FinOpsHub.psd1

```diff
@@ -0,0 +1,141 @@
+#
+# Module manifest for module 'FinOpsHub'
```

Comment:
The FinOps toolkit will include many things:
1. Hubs (the data pipeline solution)
2. Workbooks
3. Power Automate flows
4. Other ARM templates
5. Automation scripts (e.g., deploy/cleanup 1-4, other automation scripts)

Should we have 1 Toolkit module for everything or separate Hubs into its own module? Most solutions will be simple. Hubs is the only one that will have more options, but it's probably 6mo before we start to see a whole lot there. Thoughts on 1 vs. 2?

Reply:
I think one would be fine, I just got mixed up on the naming for it